### ISSUE TITLE
Support Capitalized Table Names in foreign_key field

### ISSUE THREAD

**Issue by timreichen** [TablePlus/TablePlus#1831]
1. Which driver are you using and version of it:
MariaDB 10.2

2. Which TablePlus build number are you using:
304

3. The steps to reproduce this issue:

Make a table with capitalized name for example:
```
Users
- id
```
Make a second table with a foreign key to table with capitalized name
```
Emails
- id
- user_id -> Users.id
```
TablePlus will automatically lowercase the foreign key table name, so the constraint is 
actually lowercase and also when editing the foreign key, TablePlus table select field has no selection of the table.
If system is set to allow case sensitive table names, this breaks all kinds of queries.

**Comment by huyphams** [comment]
When I create the table, MySQL returns the definition where all the ref tables are lowercase, I think this is the main reason TablePlus shows it lowercase.
<img width="709" alt="Screen Shot 2020-05-05 at 8 15 55 PM" src="https://user-images.githubusercontent.com/3654622/81070362-531d5700-8f0d-11ea-93ba-45b8cbf9c9b7.png">

**Comment by timreichen** [comment]
Yeah this is a weird and unexpected behavior and needs to be fixed. Any thoughts on this?

**Comment by huyphams** [comment]
The command was returned by MySQL, we should not fix it @timreichen it would break the compatible.
